src/ivq.tsx

```diff
@@ -243,8 +243,14 @@ export class Ivq extends KalturaPlayer.core.BasePlugin {
         });
     };
     const welcomeScreenProps: WelcomeScreenProps = {
-      welcomeMessage: this._dataManager.quizData?.welcomeMessage
+      welcomeMessage: this._dataManager.quizData?.welcomeMessage,
+      inVideoTip: this._dataManager.quizData?.inVideoTip,
     };
+    let availableAttempts = (this._dataManager.quizData?.attemptsAllowed || 0) - (this._dataManager.quizUserEntry?.version || 0);
```

Comment:
in `dataSyncManager` we have similar logic for calculation allowed attempts and qiuz version:
https://github.com/kaltura/playkit-js-ivq/blob/master/src/data-sync-manager.ts#L135
https://github.com/kaltura/playkit-js-ivq/blob/master/src/data-sync-manager.ts#L139
https://github.com/kaltura/playkit-js-ivq/blob/master/src/data-sync-manager.ts#L144
etc.
Lets create and use public methods in `dataSyncManager` for defining allowed attempts, version, etc.

Reply:
@semarche-kaltura , done.